### PR TITLE
Update ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by Shuudoushi**
I turned it into a form, instead of just instructions.

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -2,4 +2,12 @@ Thanks for wanting to bring an issue to our attention!
...
 
-If you're submitting a bug, please provide the version of CKAN you're running, your operating system, a method to re-create the bug, and any error codes generated by CKAN (if applicable).
\ No newline at end of file
+**CKAN Version:** 
+
+**Operating System:** 
+
+**The issue you are experiencing:** 
+
+**How to recreat this issue:**
```
Comment (Olympic1) [inline ISSUE_TEMPLATE.md]: recreat -> **recreate**

**Comment by Shuudoushi** [comment]
Githubs built in editor not have spell check is a pain...

**Comment by Olympic1** [comment]
Thx for the update

**Comment by plague006** [comment]
Can we drop the markup? I don't know about others but I don't want to see bolded text with every bug submission.

**Comment by Shuudoushi** [comment]
Yeah, I did it mostly just to point out to users what they should edit.

**Pull request merged**